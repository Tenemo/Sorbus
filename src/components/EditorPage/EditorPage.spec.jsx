import React from 'react';
import HomePage from './EditorPage';

describe('HomePage', () => {
    it('matches snapshot', () => {
        const wrapper = shallow(
            <HomePage />,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
