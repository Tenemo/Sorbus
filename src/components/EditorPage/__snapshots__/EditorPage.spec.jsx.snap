// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage matches snapshot 1`] = `
<section
  className="home"
>
  Editor goes here
</section>
`;
